hera_pspec/grouping.py

```diff
@@ -153,6 +154,15 @@ def average_spectra(uvp_in, blpair_groups=None, time_avg=False,
         does this for every specified key. Every stats_array key that is
         not specified is thrown out of the new averaged object.
 
+    if_error_weighted: bool, optional
+        If True, power spectra will be averaged in a way weighed by
+        error bars. Default: False
+
+    name_error_weights: string, optional
+         name_error_weights specify which kind of errors we use for weights 
```

Comment:
Should also mention that the weights are defined as $w_i = 1/\sigma_i^2$, where $\sigma_i$ is taken from the relevant field of `stats_array`.

Reply:
Sure.